src/main/java/net/torocraft/minecoprocessors/util/BookCreator.java

```diff
@@ -1,11 +1,10 @@
 package net.torocraft.minecoprocessors.util;
 
-import java.io.BufferedReader;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
```

Comment:
Does replacing the individual imports with `io.*` result in more components being imported?

I'd assume that `java.io` contains more than just `BufferedReader`, `FileNotFoundException`, `IOException`, `InputStream`, and `InputStreamReader`, and that `import java.io.*` imports all classes as well. Would there be downside to this?

Reply:
Just quickly double checked, IJ automatically replaced the single imports again with `java.io.*`, I presume it's not crucial in Java (I am no expert at all in J). Gonna push an updated version, explicit includes are generally better.